tests/bluetooth/bsim_bt/edtt_ble_test_app/hci_test_app/prj_tst.conf

```diff
@@ -3,7 +3,9 @@
 CONFIG_BT=y
 CONFIG_BT_HCI_RAW=y
 CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_BROADCASTER=y
 CONFIG_BT_CENTRAL=y
+CONFIG_BT_OBSERVER=y
```

Comment:
Why did we add this?

Reply:
In this file it needs to be removed; I'll do that